json-rpc/spec.json

```diff
@@ -196,6 +196,101 @@
 				"$ref": "#/components/contentDescriptors/GasPrice"
 			}
 		},
+		{
+			"name": "eth_feeHistory",
+			"summary": "Transaction fee history",
+			"description": "Returns transaction base fee and effective priority fee history for the requested/supported block range.",
+			"params": [
+				{
+					"name": "blockCount",
+					"description": "Number of blocks in the requested range.",
+					"required": true,
+					"schema": {
+						"$ref": "#/components/schemas/Integer"
+					}
+				},
+				{
+					"name": "lastBlock",
+					"description": "Last block number of the requested range.",
```

Comment:
Consider renaming this to "newest" instead of "last".  Depending on your frame of reference, "last" may be closer to genesis than head, or it may be closer to head than genesis.

Reply:
done